app-modules/assist-data-model/src/Models/Traits/EducatableScopes.php

```diff
@@ -34,7 +34,7 @@ public function scopeEducatableSearch(Builder $query, string $relationship, stri
             [Student::class, Prospect::class],
             fn (Builder $query, string $type) => $query->where(
                 app($type)::displayNameKey(),
-                'like',
+                'ilike',
```

Comment:
I would avoid `ilike` where possible. Using `lower()` on the column is indexable

Reply:
Changed in recent commit